### ISSUE TITLE
Increase / decrease the volume

### ISSUE THREAD

**Issue by Bazunga** [recalbox/recalbox-os#700]
Hello,

Can you add a new feature to control the sound with a hotkey combination ?

For example : 
Hotkey + Joystick UP = Increase the volume
Hotkey + Jostick BOTTOM = Decrease the volume

Thank you in advance,

And thank you for your great work.

Flow

**Comment by mgoulartBR** [comment]
+1

**Comment by marvbloke** [comment]
I have built an IKEA LACK arcade table and the ability to change volume with a hot key combination would be a great help.

**Comment by rockaddicted** [comment]
We have no more buttons free to add this feature on a "standard" controller.
We could add it on analog joystick, but that'll be usable only with "ps3/xbox" gamepad type.

Currents shortcuts : 
```
Y → Save State
X → Load State
Start → Quit
A → Reset Game
B → Menu
Up → Select Save Slot -1
Down → Select Save Slot +1
L1 → Screenshot
Right → Speedup game
Left → Rewind (if activated in options)
R2 → Next shader preset
L2 → Previous shader preset 
```

**Comment by digitalLumberjack** [comment]
Could use the joystick directions.

Le mer. 9 mars 2016 à 08:33, rockaddicted <notifications@github.com> a
écrit :

**Comment by nadenislamarre** [comment]
and this is how i did in kodi.

**Comment by rockaddicted** [comment]
Yep but it couldn't be used on a bartop.

**Comment by valvolt** [comment]
If stick directions could not be used, would it be possible to make it configurable?
Volume control would be way more useful to me than Speed-Up / Rewind ;)

**Comment by Madcore** [comment]
Why no add an option to switch some hotkeys buttons, like switch up and down from select save slot to volume? There's people that use only one saveslot and don't need to change it.

**Comment by nadenislamarre** [comment]
Personnally, i don't like this kind of complexity.

**Comment by valvolt** [comment]
Salut, is this idea still in the air or is it dead?

@rockaddicted is this list complete? If yes, this means that R1 is still free?
One idea would then to do:
R1: volume down
R2: volume up

True, R2 is already bound to 'next shader preset', but maybe L2 would be sufficient for shaders, you just would have to scroll through them instead of going back and forth?

**Comment by rockaddicted** [comment]
Nope R1 = rewind

**Comment by valvolt** [comment]
So R1 = free ? ;) *HOPE*

**Comment by rockaddicted** [comment]
Yep R1 is free, but usually, on a bartop, you don't have L2/R2 buttons, so, that don't really solve the issue.
In this case, I prefer see these shortcuts on UP/DOWN left analog stick.

**Comment by valvolt** [comment]
Aren't you the one who said that using the joystick would not work on bartops ? :)
I may be wrong but I think that the joystick on the X-Arcade is mapped to the D-Pad, meaning this solution would not work for X-Arcade users at least by default...

**Comment by rockaddicted** [comment]
Yep analog stick won't work on bartop, but your suggested solution won't work better on bartop, because there is no R2 button.
So, if we add a shortcut "gamepad only" I prefer using analog stick.

**Comment by valvolt** [comment]
It would work on mine ;)
[buttons map](https://3.bp.blogspot.com/-jwy5AN_Mt_U/VboOBS6W0NI/AAAAAAAADQo/eJe_ZYjeFjs/s1600/x-arcade%2Bingame%2B-%2Bepsxe.jpg)

**Comment by rockaddicted** [comment]
haha sure, but 99% of bartops using gpios or usb interface don't have these buttons.

**Comment by substring** [comment]
wouldn't work on my tankstick as i put hotkey on the flipper button, and Select on your R2 :stuck_out_tongue_closed_eyes:
